cypress/integration/street-save-new.spec.js

```diff
@@ -42,4 +46,19 @@ context('User Saving a New Street', () => {
     // prompt autofilled by stub above, thus changing the street name
     cy.wait('@streetPut')
   })
+
+  // this ideally would be part of its own spec test once we can get that running without a database
```

Comment:
@whitmanschorn : @louh helped me identify that the real root cause of the test I wrote not passing was the fact that Travis runs without a database. 

I tried a couple different ways of following the pattern here of mocking the request to create a street as part of my own cypress spec; I even wrote a beforeEach function in inside support/index.js so they could share that functionality. 

However, I kept running into problems where either test could run on its own, but if they both ran your test would time out. On the backend I'd see an error for an unhandled promise rejection. I tried adding some more code for trying to handle that error in our API, but honestly I was mostly guessing. 

In the end, including my spec as a condition on the existing test works. It's a quick and dirty solution, and def adds technical debt basically as soon as we need to write more cypress tests.

But it at least unblocks this particular PR. So at some point it would be great to sync up and sort out these issues, but might be best as a separate issue we can tackle later (i.e. something like 'refactor cypress tests'), with @louh 's blessing